samcli/lib/utils/subprocess_utils.py

```diff
@@ -92,10 +96,18 @@ def _print_loading_pattern():
                     # we read from subprocess stdout to avoid the deadlock process.wait function
                     # for more detail check this python bug https://bugs.python.org/issue1256
                     for line in process.stdout:
-                        decoded_line = _check_and_process_bytes(line)
+                        is_error = (
+                            not IS_NOT_WINDOWS
+                            and len(line) >= len(TERRAFORM_ERROR_PREFIX)
+                            and [c for c in line[0:4]] == TERRAFORM_ERROR_PREFIX
```

Comment:
Is this the same as 
```suggestion
                            and line[0:4] == TERRAFORM_ERROR_PREFIX
```

Reply:
No, line[0:4] will return bytes string and I wasn't able to build the same bytes string to compare